review previous portuguese docs (#17)
stack-spot/graphene-java-api-stack: logging-plugin/docs/use-case.md

## **Use Case**

> This use case is ideal to initialize Java projects with Spring Boot and gradle or maven with _logging_ support.

Reviewer: **Sugestão**
This use case is ideal to initialize Java projects with Spring Boot and _logging_ support, being able to provide the options to choose Gradle or Maven as a build tool

> It's possible to apply this plugin in an application of type APP, making use of StackSpot CLI, with the command:

Reviewer 2: ```suggestion
It's possible to apply this plugin in the app-type application. Run the STK CLI command:
```

```bash
stk apply plugin graphene-java-api-stack/logging-plugin
```

> With this the Java application with Spring Boot and gradle/maven will have support to _logging_

Reviewer 2: ```suggestion
This Java application with Spring Boot and Gradle/Maven will support _logging_.
```
